autoscale-workload-rabbit/pom.xml

```diff
@@ -56,6 +55,10 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.github.cafapi</groupId>
+            <artifactId>caf-api</artifactId>
+        </dependency>
         <dependency>
             <groupId>javax.validation</groupId>
             <artifactId>validation-api</artifactId>
```

Comment:
Why is this dependency have scope of provided?

Reply:
The provided scope is for compile time and not for runtime. There is no harm in removing provided.so it is removed.

Reply:
I think Dermot mentioned that this dependency is ok to have provided scope as you do not want this at runtime.